### ISSUE TITLE
Postgres compatibility

### ISSUE THREAD

**Issue by simhnna** [monicahq/monica#687]
#137 was closed, but monica is still not compatible. Initial migrations fail for me because "modify" is used instead of "alter"
```
 [Doctrine\DBAL\Driver\PDOException]                                        
  SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "MODIFY"  
  LINE 1: ALTER TABLE gifts MODIFY about_object_id INTEGER;                  
```

It's coming from `2017_10_14_083556_change_gift_column_structure.php`

**Comment by simhnna** [comment]
I fixed this for me using:
```
-        DB::statement('ALTER TABLE gifts MODIFY about_object_id INTEGER;');
+        DB::statement('ALTER TABLE gifts ALTER COLUMN about_object_id type INTEGER USING about_object_id::integer;');
```

**Comment by simhnna** [comment]
Next error I got:
```
  [PDOException]                                                                                                                                                                                      
  SQLSTATE[2BP01]: Dependent objects still exist: 7 ERROR:  cannot drop index unique_for_each_account_email_pair because constraint unique_for_each_account_email_pair on table contacts requires it  
  HINT:  You can drop constraint unique_for_each_account_email_pair on table contacts instead.
```

**Comment by Vici37** [comment]
I was able to get past this error by changing `2017_11_10_204035_delete_contact_fields_from_contacts.php`

```
-            $table->dropIndex('unique_for_each_account_email_pair');
+            $table->dropUnique('unique_for_each_account_email_pair');
```

**Comment by asbiin** [comment]
Migrations with postgres was fixed with #785.
I will not close this issue because (a lot of) other errors happens with Postgres right now ...

**Comment by asbiin** [comment]
This one is a duplicate of #137
